components/merkledb/src/entry.rs

```diff
@@ -193,33 +191,3 @@ where
         previous
     }
 }
-
-impl<T, V> ObjectHash for Entry<T, V>
```

Comment:
Can't add the comment to the corresponding line, but I think that `ObjectHash` restriction should be removed from

```rust
impl<T, V> Entry<T, V>
where
    T: RawAccessMut,
    V: BinaryValue + ObjectHash,
```

By the way it's funny that we had no tests for this.